jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/inputs/transferlist/TransferList.java

```diff
@@ -15,104 +15,220 @@
+ * to the left or right list. You can select some items and move them or
+ * move all items at once.
+ *
+ * @see <a href="https://mui.com/components/transfer-list/">Transfer List MUI documentation</a>
+ * @see <a href="https://jdi-testing.github.io/jdi-light/material">MUI test page</a>
  */
-
 public class TransferList extends UIBaseElement<TransferListAssert> implements ISetup {
 
+    /**
+     * Locator for item checkbox.
+     */
     private String itemCheckbox;
+
+    /**
+     * Locator for all left list items.
+     */
     private String leftListItems;
+
+    /**
+     * Locator for all right list items.
+     */
     private String rightListItems;
 
+    /**
+     * Locator for button which moves selected items to the right list.
+     */
     String moveRightButton;
+
+    /**
+     * Locator for button which moves selected items to the left list.
+     */
     String moveLeftButton;
+
+    /**
+     * Locator for Simple Transfer List button which moves all items to the right list.
+     */
     String moveAllRightButton;
+
+    /**
+     * Locator for Simple Transfer List button which moves all items to the left list.
+     */
     String moveAllLeftButton;
+
+    /**
+     * Locator for Enhanced Transfer List left card header checkbox which selects all items.
+     */
     String allItemsLeftCheckbox;
+
+    /**
+     * Locator for Enhanced Transfer List right card header checkbox which selects all items.
+     */
     String allItemsRightCheckbox;
 
+    /**
+     * Texts of left items for update left list items method.
+     */
     private CacheAll<List<String>> leftItems = new CacheAll<>(ArrayList::new);
+
+    /**
+     * Texts of right items for update right list items method.
+     */
     private CacheAll<List<String>> rightItems = new CacheAll<>(ArrayList::new);
 
+    /**
+     * Updates left items list after moving and returns updated actual items list.
+     *
+     * @return updated items of this list as {@link List}
+     */
+    @JDIAction("Update '{name}' left items")
     public List<String> updateLeftItems() {
-        return leftItems.set(this.finds(leftListItems).stream().map(UIElement::getText).collect(
+        return leftItems.set(core().finds(leftListItems).stream().map(UIElement::getText).collect(
                 Collectors.toList()));
     }
 
+    /**
+     * Updates right items list after moving and returns updated actual items list.
+     *
+     * @return updated items of this list as {@link List}
+     */
+    @JDIAction("Update '{name}' right items")
     public List<String> updateRightItems() {
-        return rightItems.set(this.finds(rightListItems).stream().map(UIElement::getText).collect(
+        return rightItems.set(core().finds(rightListItems).stream().map(UIElement::getText).collect(
                 Collectors.toList()));
     }
 
-    @JDIAction("Checks {name}'s '{0}' item")
+    /**
+     * Checks checkbox of the specific item using its text (full equality is used by searching).
+     *
```

Comment:
Checks checkbox of the item with given text (full equality is used by searching). If it's already checked, does nothing.